tasks/run_claim_review_parser.rb

```diff
@@ -34,13 +34,24 @@ def self.requeue(service)
     false
   end
 
+  def self.not_enqueued_anywhere_else(service)
+    [Sidekiq::RetrySet, Sidekiq::ScheduledSet, Sidekiq::Queue].each do |queue|
+      queue.new.each do |job|
+        if job.item["args"] && job.item["args"][0] == service.to_s
+          return false
+        end
+      end
```

Comment:
Just noting that this can get inefficient and lead to memory issues in Sidekiq, but I don't think the Fetch queue gets big enough for this (specially after the fix introduced in this PR is in place), so I think we're good... @DGaffney please confirm.